### PR TITLE
Rbac

### PR DESCRIPTION
**Pull request by pb82**
Adds the rbac resources to allow `oc` to be used inside the backup containers using a service account. Unfortunately adding those resources directly to the template does not work because creating the jobs would then fail if they already existed.

Verification steps:

1. run `oc apply -f templates/openshift/rbac`
1. Try using oc in your backup script
1. Create the job using for example:
```
oc new-app \
-f backup-job-template.yaml \
-p 'COMPONENT=mysql' \
-p 'COMPONENT_SECRET_NAME=sample-mysql-secret' \
-p 'BACKEND_SECRET_NAME=sample-s3-secret' \
-p 'ENCRYPTION_SECRET_NAME=sample-gpg-secret' \
-p 'IMAGE=<your dev image>' \
-p 'NAME=resources-backup' -n webapp
```
4. It should work

# Notes

If you need permissions to other resources than configmaps or secrets, feel free to add them to the clusterrole.

### CODE REVIEW

File: templates/openshift/backup-cronjob-template.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ objects:
               labels:
                 cronjob-name: ${NAME}
             spec:
+              serviceAccountName: backupjob
```
Comment (odra) [inline templates/openshift/backup-cronjob-template.yaml]: Could this be a param that defaults to "backupjob"?

Comment (pb82) [reply templates/openshift/backup-cronjob-template.yaml]: sure, will update

File: image/Dockerfile [hunk 2]
```diff
@@ -13,7 +13,8 @@ RUN yum --enablerepo=extras install -y epel-release && \
 COPY tools /opt/intly/tools
 RUN chown -R 1001:root /opt/intly
 
-RUN find /opt/intly/tools -type f -exec chmod +x {} \;
+RUN find /opt/intly/tools -type f -exec chmod +x {} \; && \
+    mkdir /.kube && touch /.kube/config && chmod -R 777 /.kube
```
Comment (odra) [inline image/Dockerfile]: does it need to be that permissive? does applying the above config `chown -R 1001:root` for the .kube folder fail?

Comment (pb82) [reply image/Dockerfile]: Yes, even just `744` isn't enough. I'll see if there is some other way to make it less permissive.

Comment (odra) [reply image/Dockerfile]: what about 774? In theory just the group and the owner should need to write files in there

Comment (pb82) [reply image/Dockerfile]: 774 works, updated.

**Review by odra — APPROVED** [review]

**Pull request merged**